src/Phinx/Console/Command/Create.php

```diff
@@ -50,6 +58,13 @@ protected function configure()
                 PHP_EOL,
                 PHP_EOL
             ));
+
+        // An alternative template.
+        $this->addOption('template', 't', InputOption::VALUE_REQUIRED, 'Use an alternative template');
+
+        // A classname to be used to gain access to the template content as well as the ability to
+        // have a callback once the migration file has been created.
+        $this->addOption('class', 'l', InputOption::VALUE_REQUIRED, 'Use a class implementing "' . self::CREATION_INTERFACE . '" to generate the template');
```

Comment:
I don't know how GitHub is handling things, but I've got 2 feature branches open - one for each issue. But 1 repo for all the work.

I use SourceTree with git-flow.

Not sure what I should be doing to keep these separate. Seems Github deals with repos and not branches when it comes to pull requests. So, does that mean I need multiple repos to have multiple pull requests? Please advise.